pages/teams.html

```diff
@@ -167,8 +165,7 @@ <h4 class="title" id="editor">Editor • <a href="https://chat.godotengine.org/c
 	</div>
 
 	<div class="teams-team-section">
-		<h4 class="title" id="scripting">Scripting • <a href="https://chat.godotengine.org/channel/scripting">#scripting</a>
-		</h4>
+		<h4 class="title" id="scripting">Scripting</h4>
```

Comment:
Oops, it doesn't seem that the change to ".NET" was committed.

Reply:
Should be below those lines, if I'm not missing anything.

Reply:
Yeah maybe, guess we can do followup work then though :) And no worries